experiment/resultstore/main.go

```diff
@@ -107,59 +122,123 @@ func run(opt options) error {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 	defer cancel()
 
-	result, err := download(ctx, opt)
+	var gcsAccount string
+	if opt.gcsAuth {
+		gcsAccount = opt.account
+	}
+	storageClient, err := storageClient(ctx, gcsAccount)
+	if err != nil {
+		return fmt.Errorf("storage client: %v", err)
+	}
+
+	fmt.Println("Reading testgrid config...")
+	cfg, err := config.Read(opt.testgridConfig, ctx, storageClient)
 	if err != nil {
-		return fmt.Errorf("download %s: %v", opt.path, err)
+		return fmt.Errorf("read testgric config: %v", err)
 	}
 
-	inv, target, test := convert(
-		opt.project,
-		"hello again",
-		opt.path,
-		*result,
-	)
-	print(inv.To(), test.To())
+	var rsClient *resultstore.Client
+	if opt.project != "" {
+		rsClient, err = resultstoreClient(ctx, opt.account, resultstore.Secret(opt.secret))
+		if err != nil {
+			return fmt.Errorf("resultstore client: %v", err)
+		}
+	}
 
-	if opt.project == "" {
-		return nil
+	if opt.path.Bucket() != "" { // transfer just a single, specific build
```

Comment:
tiny nit: at a glance this looks like it's testing specifically the bucket, rather than whether `path` is set at all. It might be clearer if it checked the path rather than the bucket?

Reply:
If someone does `--build=gs://bucket` I want it to error, rather than transfer all jobs.